.github/docs/wiki/Setting-up.md

```diff
@@ -17,15 +17,23 @@ sudo apt install -y docker docker-compose
 ## configuring bancho.py
 
 all configuration for the osu! server (bancho.py) itself can be done from the
-`.env` file. we provide an example `.env.example` file which you can use as a base.
+`.env` and `logging.yaml` files. we will provide example files for each, which
+you can use as a base and modify as you'd like.
 
 ```sh
 # create a configuration file from the sample provided
 cp .env.example .env
 
+# create a logging configuration file from the sample provided
+cp logging.yaml.example logging.yaml
```

Comment:
Any reason to have this as an extra setup step if we don't expect users to edit it? Could we just provide the yaml file?

Reply:
I see the comment now. Sorry, on shitty wifi